start-valheim-server.sh

```diff
@@ -39,7 +39,15 @@ echo "Valheim port is: $VALHEIM_PORT"
 echo "Valheim server name is: $VALHEIM_SERVER_NAME"
 echo "Valheim world name is: $VALHEIM_WORLD_NAME"
 
+steamcmd/steamcmd.sh +login anonymous \
++force_install_dir $VALHEIM_SERVER_DIR \
++app_update 896660 \
+validate +exit > "/home/steam/valheim-data/steamcmd_log.txt"
+
 cd $VALHEIM_SERVER_DIR
+
+echo ./valheim_server.x86_64 -name $VALHEIM_SERVER_NAME -port $VALHEIM_PORT -world $VALHEIM_WORLD_NAME -password $VALHEIM_PASSWORD -savedir $VALHEIM_DATA_DIR > "/home/steam/valheim-data/call.txt"
+
```

Comment:
It looks like this line is just saving the whole command string to run the server to that text file, but is not actually running the command?  

Mind clarifying here, and also adding an inline comment explaining what this line is doing and its purpose?

Reply:
yes you are right.
I did this for debugging purposes to see, what is executed when something does not work.
If you run it with docker compose (maybe even detached), you need to see what was going on.